src/s3/args.rs

```diff
@@ -1116,21 +1116,33 @@ pub struct ListObjectsArgs<'a> {
     pub extra_headers: Option<&'a Multimap>,
     pub extra_query_params: Option<&'a Multimap>,
     pub region: Option<&'a str>,
+    /// Bucket to list
```

Comment:
This is obvious argument to to list objects API. Either remove the comment or add appropriately like `To fetch list of objects on this bucket`